Harmonize javascript 6 (#85)
kuzzleio/documentation-V2: src/sdk-reference/js/6/collection/update-specifications/index.md

# updateSpecifications

> The updateSpecifications method allows you to create or update the validation specifications for a `<collection>`.

Reviewer: ```suggestion
The updateSpecifications method allows you to create or update the validation specifications of `<collection>`.
```
